samples/javascript_nodejs/14.nlp-with-dispatch/.env

```diff
@@ -1,2 +1,18 @@
 MicrosoftAppId=
 MicrosoftAppPassword=
+
+Luis-Home-Automation-AppId=
```

Comment:
We should be consistent in how we use keys here.  With some keys were are using PascalCase which is the pattern, and with other keys you're using kabob-case.  Please use PascalCase for Luis keys and QnA keys, and NlpWithDispath... keys

Reply:
PascalCase it is! This original in the C# version made my soul cry: "`Luis-nlp-with-dispatchDispatch-Authoringkey`"

Not only that, but 1. js doesn't like "with" (reserved keyword) and 2. node/.env doesn't like `-` in `.env` files apparently

Removed `-`'s and switched to Pascal :thumbsup: